drivers/overlay/overlay.go

```diff
@@ -1216,11 +1216,7 @@ func (d *Driver) recreateSymlinks() error {
 	}
 	linksDir := filepath.Join(d.home, "l")
 	// This makes the link directory if it doesn't exist
-	rootUID, rootGID, err := idtools.GetRootUIDGID(d.uidMaps, d.gidMaps)
-	if err != nil {
-		return err
-	}
-	if err := idtools.MkdirAllAs(path.Join(d.home, linkDir), 0700, rootUID, rootGID); err != nil {
+	if err := idtools.MkdirAllAs(path.Join(d.home, linkDir), 0755, 0, 0); err != nil {
```

Comment:
I don't know nothing about nothing, but line 1217 made me sweat. Since `linksDir` (plural) isn't actually used until line 1259 (far, far below), and since a casual monkey reader like me could easily think that the singular `linkDir` right here could be a typo or misused variable... would you consider doing a little cleanup while you're here, moving the `linksDir` plural definition way way down, and adding a comment explaining how it differs from singular-`linkDir`?

Reply:
I've not noticed it, thanks for the hint.  Pushed a new patch